app/blueprints/api/v4/race_blueprint.rb

```diff
@@ -0,0 +1,26 @@
+class Api::V4::RaceBlueprint < Blueprinter::Base
+  fields :id, :status, :visibility, :notes, :started_at, :created_at, :updated_at
+
+  field :type do |race, _|
```

Comment:
Oh interesting.  I always just went with the `_` because rubocop yells at you to do that if you have an unused block parameter.  I'll give that a shot.

Reply:
Ohh ok, I didn't realize Rubocop yelled about it. Feel free to disregard what I said then.